a3p-integration/proposals/a:upgrade-14/invite-submission/sendInvite.tpl

```diff
@@ -0,0 +1,29 @@
+// to be replaced before execution
+const addr = 'XX_ADDRESS_XX';
```

Comment:
while we probably don't want to add a dependency for this, it would be good to choose a templating syntax. I suggest [EJS ](https://ejs.co/) which is supported by Lodash https://lodash.com/docs/4.17.15#template

```suggestion
const addr = <%= address %>;
```

We don't have to import either of those libs--you can do a simple string replace in this case--but at least the syntax of the transformation would have clear semantics. And this file would have a `.ejs` extension.

Reply:
Second thought, that syntax is very geared to HTML. I'm surprised there's no library for rendering from a JS template literal defined in a file. Maybe too trivial. The biggest lift is syntax highlighting for IDEs.

Reply:
I actually wrote such a library 10 years ago, very non-trivial

Reply:
Much harder before JS template literals. I had in mind something like dcce084d0. But now that I see it, there's not much virtue over EJS. EJS works fine and what I really want is for my IDE to highlight it correctly (not assume HTML).